platform/kube/controller.go

```diff
@@ -33,6 +33,9 @@ import (
 	"istio.io/pilot/model"
 )
 
+// NodeAZLabel is the well-known label for kubernetes node AZs
```

Comment:
What's the difference? I went with labels based on the definitions of the two in the docs.